grpc/proto/batch.proto

```diff
@@ -43,8 +41,8 @@ message BatchObject {
   // protolint:disable:next REPEATED_FIELD_NAMES_PLURALIZED
   repeated float vector = 2;
   Properties properties = 3;
-  string class_name = 4;
-  string tenant=5;
+  string collection = 4;
+  string tenant = 5;
 }
 
 message BatchObjectsReply {
```

Comment:
Can we add to `BatchObjectsReply` status field? I know that we are not returning any results in case whole batch succeeds to be saved in DB, but al least a status would be helpful to determine if all is OK or we should look for errors

Reply:
hmmm, how about we rename 

```
repeated BatchResults results = 2;
```
to
```
repeated BatchErrors errors = 2;
```

the status field would be the same as `if len(result.errors) > 0`, which seems pretty clear to me

Reply:
hmm, we can do this (introduce `BatchErrors`) instead of `results` which is empty if successful batch import, but I would still add an explicit status field